src/main/java/seedu/address/logic/interview/AssignInterviewCommand.java

```diff
@@ -33,6 +33,8 @@
     public static final String MESSAGE_SUCCESS = "Candidates added to interview %1$s: %2$s";
     public static final String MESSAGE_CANDIDATE_DID_NOT_APPLY = "Candidate %1$s (%2$s) did not apply for "
             + "this position: %3$s";
+    public static final String MESSAGE_CANDIDATE_HAS_INTERVIEW = "The candidate(s) has/have already been assigned"
+            + " to this interview.";
```

Comment:
Maybe feedback to the user which candidate has been assigned? Like use the MESSAGE_CANDIDATE_DID_NOT_APPLY as example.

Reply:
resolved